### PR TITLE
Added 2 puns 😄

### PR DESCRIPTION
**Pull request by AzrizHaziq**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -190,5 +190,13 @@
...
     "pun": "Q: Why were the Javascript plumbers delayed?",
     "punchline": "A: Because they had to await async."
+  },
+  {
+    "pun": "Q: Why do Java developer wear glasses?",
+    "punchline": "A: Because they don't C#!"
+  },
+  {
+    "pun": "Q: How do you generate a random string?",
+    "punchline": "Put a Windows user in front of vim, and tell them to exit"
```
Comment (AlexLakatos) [inline puns.json]:
```suggestion
    "punchline": "Put a Windows user in front of vim, and tell them to exit!"
```

File: puns.json [hunk 2]
```diff
@@ -190,5 +190,13 @@
   {
     "pun": "Q: Why were the Javascript plumbers delayed?",
     "punchline": "A: Because they had to await async."
+  },
+  {
+    "pun": "Q: Why do Java developer wear glasses?",
+    "punchline": "A: Because they don't C#!"
+  },
```
Comment (AlexLakatos) [inline puns.json]: This one is already in the list, please remove it.
https://github.com/AlexLakatos/computer-puns/blob/a75fef11c16b034fe812acc8d8157365ddd0927a/puns.json#L71

**Comment by AzrizHaziq** [comment]
Hi @AlexLakatos , done with the new changes. 😄

**Comment by AlexLakatos** [comment]
Thanks! Happy Hacktoberfest!

**Pull request merged**